File: Linguagem C++/Estrutura Repetitiva/MediaPonderada.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    double n1,n2,n3,media;

    cout << "Quantos casos voce vai digitar? ";
    cin >> n;

    for (int i=1;i<=n;i++){
        cout << "Digite tres numeros:" << endl;
        cin >> n1;
        cin >> n2;
        cin >> n3;
        media = (n1*2+n2*3+n3*5) / 10;
        cout << fixed << setprecision(1);
        cout << "MEDIA = " << media << endl;
    };

    return 0;
}

File: Linguagem Java/Exercicios/MediaPonderada.java
import java.util.Locale;
import java.util.Scanner;

public class MediaPonderada {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
         
		int n;
		double n1,n2,n3,media;
		
		System.out.print("Quantos casos voce vai digitar? ");
	    n = sc.nextInt();
	    
	    for (int i=1;i<=n;i++) {
	    	System.out.println("Digite tres numeros: ");
	    	n1 = sc.nextDouble();
	    	n2 = sc.nextDouble();
	    	n3 = sc.nextDouble();
	    	
	    	media = (n1*2+n2*3+n3*5) / 10;
	    	System.out.println("MEDIA = "+String.format("%.1f", media));
	    };
	
	    sc.close();
	}

}
